ee-9/source-transform/webservices/server-integration/pom.xml

```diff
@@ -0,0 +1,212 @@
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2010, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly</groupId>
+        <artifactId>wildfly-ee-9-source-transform-parent</artifactId>
+        <!--
+        Maintain separation between the artifact id and the version to help prevent
+        merge conflicts between commits changing the GA and those changing the V.
+        -->
+        <version>26.0.0.Beta1-SNAPSHOT</version>
+       <relativePath>../../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>wildfly-webservices-server-integration-jakarta</artifactId>
+    <packaging>jar</packaging>
+
+    <name>WildFly: Web Services Server Integration Subsystem (Jakarta Namespace)</name>
+
+   <properties>
+      <transformer-input-dir>${project.basedir}/../../../../webservices/server-integration</transformer-input-dir>
+      <jakarta.xml.ws-api.version>3.0.1</jakarta.xml.ws-api.version>
+      <jbossws.common.version>4.0.0.Final</jbossws.common.version>
+      <jbossws-spi.version>4.0.0.Final</jbossws-spi.version>
+   </properties>
+
+    <dependencies>
+       <dependency>
+          <groupId>jakarta.annotation</groupId>
+          <artifactId>jakarta.annotation-api</artifactId>
+       </dependency>
+
+       <dependency>
+          <groupId>jakarta.servlet</groupId>
+          <artifactId>jakarta.servlet-api</artifactId>
+      </dependency>
+
+       <dependency>
+          <groupId>jakarta.ejb</groupId>
+          <artifactId>jakarta.ejb-api</artifactId>
+       </dependency>
+
+       <dependency>
+          <groupId>jakarta.xml.ws</groupId>
+          <artifactId>jakarta.xml.ws-api</artifactId>
+          <version>${jakarta.xml.ws-api.version}</version>
```

Comment:
We should use this artifact: 
```
<dependency>
    <groupId>org.jboss.spec.jakarta.xml.ws</groupId>
    <artifactId>jboss-jakarta-xml-ws-api_3.0_spec</artifactId>
    <version>1.0.0.Final</version>
</dependency>
```
And add this to parent pom dependency management too.